report/renderer/category-renderer.js

```diff
@@ -171,13 +171,14 @@ export class CategoryRenderer {
   /**
    * @param {LH.ReportResult.Category} category
    * @param {Record<string, LH.Result.ReportGroup>} groupDefinitions
+   * @param {{gatherMode: LH.Result.GatherMode}=} options
```

Comment:
better as 
```suggestion
   * @param {{displayMode: 'gauge'|'fraction'}}=} options
```
(or whatever option name)?

Reply:
We might want to adjust the display mode based on gather mode and category at some point. Are you saying we should do this only for this function?

Reply:
I meant for all of `render`/`renderCategoryHeader`/`renderCategoryScore` and have `reportRenderer.render()` decide which option to pass in for how to display the score, but we can also defer if you'd prefer (they're equivalent for now). Whatever you think works/is easier to move forward with.